packages/volto/news/5518.feature

```diff
@@ -0,0 +1 @@
+Put cloneData function for slate block in order to change tu uuids of fragments when copy @dobr1408
```

Comment:
I'm not sure what "tu" is supposed to be. Is that a typo?

Based on what I read in the discussion, and what little I can decipher in the code, here is what I think is the intent. Please correct as needed.

```suggestion
When copying a text block—implemented in Slate—using the sidebar that contains UUIDs attached to elements in the data, the UUIDs get copied into the new block, making them not unique. Now instead of duplicating the UUIDs, a unique nanoid will be generated and assigned to the copy, preserving uniqueness. @dobr1408
```

Reply:
Also if this fixes a bug, and does not introduce a new feature, then the file should be renamed to `5518.bugfix`.